srcpkgs/devilution/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'devilutionx'
+pkgname=devilutionx
+version=1.0.0
+revision=1
+wrksrc="devilutionX-${version}"
+build_style=cmake
+configure_args="-DBINARY_RELEASE=ON -DTTF_FONT_PATH=\"/usr/share/fonts/truetype/CharisSILB.ttf\""
```

Comment:
Could use single-quotes and avoid this, I think.

Reply:
I am using exactly, which the author suggests and used in Fedora and Debian.

Reply:
Ok.